README.md

```diff
@@ -1,78 +1,65 @@
-# Grafana Plugin Repository
+# Community plugins
 
-This repository lists all officially supported Grafana plugins. Plugins in this repository are listed on [Grafana.com](https://grafana.com/grafana/plugins), and can be [installed](https://grafana.com/docs/grafana/latest/plugins/installation/) locally using [Grafana CLI](https://grafana.com/docs/grafana/latest/administration/cli/#plugins-commands) or by [Grafana Cloud](https://grafana.com/products/cloud/) users.
+We created this repository in 2016 to accept plugin submissions from the Grafana community. 5 years and more than 150 published plugins later, it's clear that community plugins have been instrumental in the success of Grafana. We'd like to make the process of submitting plugins even easier.
 
-We review all plugins before they are published. This means that it may take some time before we can review your plugin.
+We're excited to announce that you can now [submit your plugin](#submit-your-plugin) and [maintain your plugin](#maintain-your-plugin) directly from your Grafana Cloud account!
```

Comment:
Again, not totally sure if "marketing" language is intentional/appropriate...this is kind of a grey area. 
However, terms like "can now" don't age well. 

I would attach this sentence to the previous paragraph (line three), "You can now submit and maintain plugins directly from your Grafana Cloud account."